inlang/source-code/env-variables/build.ts

```diff
@@ -17,16 +16,11 @@ config({ path: rootEnvFilePath, override: false })
 const { error: errors } = validateEnvVariables({ forProduction: false })
 
 if (errors) {
-	console.error("💡 Some env variables are not defined. Fetching public env variables remotely...")
+	console.error(
+		"❗️❗️ Some env variables are not defined or do not pass validation. If you are an external developer, please run `npm run fetchPublicEnv` to get the latest version, if you are part of the inlang team, please check your doppler setup."
```

Comment:
`npm run fetchPublicEnv` will break in the monorep setup, assuming that developers start from the root of the repo. 

`npm run fetchPublicEnv -w @inlang/env-variables` will likely

Reply:
@samuelstroschein  i added the command to the root of the repo too:
"fetchPublicEnv": "npx nx run @inlang/env-variables:fetchPublicEnv",